docs/source/docker/advanced/active_learning.rst

```diff
@@ -176,12 +188,18 @@ Your embeddings_al.csv should look similar to this:
 +----------------+--------------+--------------+--------------+--------------+---------+-----------+
 
 
-Step 3. Use the Lightly docker to perform a sampling on the scores.
----------------
-Run the docker and use the generated embedding file from the last step.
-Then perform an active learning sampling using the `CORAL` sampler.
-E.g. use the following bash script.
+Run Active Learning using the Docker
+---------------------------------------
+
+At this point you should have an `embeddings.csv` file with the active learning 
+scores in a column. 
 
+We can now perform an active learning sampling using the `CORAL` sampler. In order
+to do the sampling on the `embeddings.csv` file we need to make this file
+accessible to the docker. An easy way is to use the `shared_dir` feature of the
```

Comment:
It's the only way at the moment so I don't konw if saying "An easy way ..." makes sense. What about "We can do this by using the `shared_dir` feature of the docker..."